### PR TITLE
CSS tweaks + Sharing

### PR DESCRIPTION
**Pull request by lukechampine**
Text is more readable now, and the share dialog has been restored.

### CODE REVIEW

**Comment by DavidVorick** [comment]
devtools are still active for the wallet plugin

**Comment by DavidVorick** [comment]
I find the 'MaxFilesize' field to be confusing. With the new revision structure, does the host even pay attention to that field? And why is the default 1GB? (I think I fixed that in my open PR)

File: app/plugins/Files/js/buttons.js [hunk 1]
```diff
@@ -47,16 +50,25 @@ eID('upload-choice').onclick = function() {
...
+		// be refocused after the dialog box is closed.
+		eID('nickname-file-input').focus();
 	}
 };
+eID('nickname-file-input').addEventListener("keydown", function(e) {
+    e = e || window.event;
+    if (e.keyCode == 13) {
+        eID('upload-file').click();
+    }
+}, false);
```
Comment (DavidVorick) [inline app/plugins/Files/js/buttons.js:67]: What does this code do, can you add a comment to it?

File: app/plugins/Files/js/buttons.js [hunk 2]
```diff
@@ -82,17 +94,35 @@ eID('add-sia-file').onclick = function() {
...
 	show('paste-ascii');
 	show('add-ascii-file');
+	eID('paste-ascii-input').focus();
 };
+eID('paste-ascii-input').addEventListener("keydown", function(e) {
+    e = e || window.event;
+    if (e.keyCode == 13) {
+        eID('add-ascii-file').click();
+    }
+}, false);
```
Comment (DavidVorick) [inline app/plugins/Files/js/buttons.js:113]: same here

File: app/plugins/Wallet/js/lifecycle.js [hunk 3]
```diff
@@ -82,7 +82,7 @@ addResultListener('update-history', function(result) {
 // Called upon showing
 function start() {
 	// DEVTOOL: uncomment to bring up devtools on plugin view
-	// IPC.sendToHost('devtools');
+	 IPC.sendToHost('devtools');
```
Comment (DavidVorick) [inline app/plugins/Wallet/js/lifecycle.js]: comment back out

File: app/plugins/Files/js/buttons.js (hunk 2 above)
Comment (lukechampine) [inline app/plugins/Files/js/buttons.js:113]: fixed

**Pull request merged**